.mci.yml

```diff
@@ -282,6 +282,9 @@ pre:
     params:
       script: |
         rm -rf "mongo-cxx-driver"
+        if [ ! -d /data/db ]; then
+            mkdir /data/db
+        fi
```

Comment:
Maybe be a bit more defensive?
`
if [ ! -e /data/db ]; then
    mkdir /data/db
elif [ ! -d /data/db ]; then
    # raise an error that it exists but isn't a dir
fi
`